shaper-core/src/main/kotlin/dev/icerock/tools/shaper/core/HandlebarsFactory.kt

```diff
@@ -6,10 +6,16 @@ package dev.icerock.tools.shaper.core
 
 import com.github.jknack.handlebars.Handlebars
 import com.github.jknack.handlebars.Helper
+import com.github.jknack.handlebars.io.FileTemplateLoader
+import java.io.File
+import java.nio.file.Paths
 
 object HandlebarsFactory {
-    fun create(): Handlebars {
-        val handlebars = Handlebars()
+    fun create(includePath: String = "_includes"): Handlebars {
+        val partialDir = File("${Paths.get("").toAbsolutePath()}/$includePath")
+        partialDir.mkdirs()
+        val handlebars = Handlebars(FileTemplateLoader(partialDir, ".hbs"))
```

Comment:
will be this work if includePath was not in work directory? we support search of template in multiple templates directories at now and this should work with any template dir